src/lib/utils/misc.ts

```diff
@@ -32,10 +32,9 @@ export const inviteOptions: InviteGenerationOptions = {
 		PermissionFlagsBits.EmbedLinks,
 	]),
 };
-export const invite = container.client.generateInvite(inviteOptions);
 export const InviteButton = new ButtonBuilder()
 	.setStyle(ButtonStyle.Link)
-	.setURL(invite)
+	.setURL(container.clientInvite)
```

Comment:
This will stay as undefined, because you import it in the client before calling login), so now ready will not log anything and the button won't have the URL either